sdk/cosmos/azure-cosmos-kafka-connect/doc/configuration-reference.md

```diff
@@ -2,64 +2,64 @@
-| `kafka.connect.cosmos.source.changeFeed.maxItemCountHint` | `1000`                   | The maximum number of documents returned in a single change feed request. But the number of items received might be higher than the specified value if multiple items are changed by the same transaction.                                           |
-| `kafka.connect.cosmos.source.metadata.poll.delay.ms`      | `300000`                 | Indicates how often to check the metadata changes (including container split/merge, adding/removing/recreated containers). When changes are detected, it will reconfigure the tasks. Default is 5 minutes.                                           |
-| `kafka.connect.cosmos.source.metadata.storage.topic`      | `_cosmos.metadata.topic` | The name of the topic where the metadata are stored. The metadata topic will be created if it does not already exist, else it will use the pre-created topic.                                                                                        |
-| `kafka.connect.cosmos.source.messageKey.enabled`          | `true`                   | Whether to set the kafka record message key.                                                                                                                                                                                                         |
-| `kafka.connect.cosmos.source.messageKey.field`            | `id`                     | The field to use as the message key.                                                                                                                                                                                                                 |
+| `azure.cosmos.source.database.name`               | None                     | Cosmos DB database name.                                                                                                                                                                                                                             |
+| `azure.cosmos.source.containers.includeAll`       | `false`                  | Flag to indicate whether reading from all containers.                                                                                                                                                                                                |
+| `azure.cosmos.source.containers.includedList`     | `[]`                       | Containers included. This config will be ignored if azure.cosmos.source.includeAllContainers is true.                                                                                                                                        |
```

Comment:
includeAllContainers is not a valid property name anymore :)

Reply:
updated